### DISCUSSION TITLE
Scale wreck commonness to difficulty

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#5301] [Ideas and suggestions]
Have wreck commonness (how often they appear) scale to difficulty, I think it would be better than scaling its loot pools to difficulty.

It would make sure that wrecks are not guaranteed to spawn every level, while still retaining that "value" when you find a wreck in low difficulty levels (if you're lucky enough to come across one).

And it would make sense in a way.
High difficulty = Higher sub casualties = More often wrecks = More potential loot
Opposite for low difficulty

This should be another way to enhance the "pull" / attraction factor to go into harder regions in search of more loot in campaign.

**Comment by ubertpendragon** [comment]
It's not just frequency, but also a symptom of only having 4 relatively large wrecks so far. 
Diluting the overall wreck pool with smaller wrecks would make wreck spawns less rewarding.

Having Small/Medium/Large wreck categories as well, pushing larger wrecks towards harder areas

See: cutting Dugong wreck in 7 parts
![Dugongwreck](https://user-images.githubusercontent.com/73229309/112356864-49d8dc80-8ccf-11eb-806a-dff305337a1b.jpg)

**Comment by Udrakan** [comment]
autofill loot tables as well as the amount of storage on the wrecks contributes in a major way (if not the biggest way) to players being able to total traders after 1-2 levels, as well as wrecks becoming inventory management game because the enemies of this loot are not scaled to difficulty or the size of wreck at all.
